frontend/.env.dist

```diff
@@ -1,6 +1,5 @@
 LOGIN_API_URL=http://localhost/login_api/
-COMMUNITY_API_STATE_BALANCE_URL=http://localhost/state-balances/
-COMMUNITY_API_TRANSACTION_CREATION_URL=http://localhost/transaction-creations/
-COMMUNITY_API_TRANSACTION_SEND_COINS=http://localhost/api
-VUE_PATH=/vue
-APP_VERSION=0
\ No newline at end of file
+COMMUNITY_API_STATE_BALANCE_URL=http://localhost/api/
+COMMUNITY_API_TRANSACTION_CREATION_URL=http://localhost/api/
+COMMUNITY_API_TRANSACTION_SEND_COINS=http://localhost/api/
```

Comment:
```suggestion
COMMUNITY_API_URL=http://localhost/api/
```
This file seems to have line ending problems or for some other reason do not allow proper comparison diffs

⚠️  Accepting this change would require some more adjustments not noted here as suggestions